junit-trait-runner/pom.xml

```diff
@@ -27,6 +27,7 @@
 
     <properties>
         <maven.deploy.skip>true</maven.deploy.skip>
+        <sonar.skip>true</sonar.skip>
```

Comment:
Any idea why this modules shows as uncovered? It does run as part of the build, but maybe the fact that it's invoked by JUnit code through reflection prevents coverage information from being gathered.

Anyway it's two classes in a tangential modules. I'll go ahead and merge.